lib/fs.js

```diff
@@ -38,13 +38,26 @@ const Stream = require('stream').Stream;
 const EventEmitter = require('events');
 const FSReqWrap = binding.FSReqWrap;
 const FSEvent = process.binding('fs_event_wrap').FSEvent;
-const internalFS = require('internal/fs');
 const internalURL = require('internal/url');
 const internalUtil = require('internal/util');
-const assertEncoding = internalFS.assertEncoding;
-const stringToFlags = internalFS.stringToFlags;
 const getPathFromURL = internalURL.getPathFromURL;
 
+const {
```

Comment:
What is the issue with accessing `internalFs.` For example `realpathCacheKey` is only used in one location, why create a module level variable to manage instead of being explicit and accessing it from `internalFs` the one time?